### PR TITLE
Remove case_search_home_page feature toggle

### PR DESCRIPTION
**Pull request by Sjones352**
Connects #9590

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -168,7 +168,7 @@ def no_cache
   end
 
   def case_search_home_page
-    if feature_enabled?(:case_search_home_page)
+    if current_user&.authenticated?
```
Comment (Sjones352) [inline app/controllers/application_controller.rb]: By removing the feature toggle, I implemented a method to check if user is authenticated. I am guessing by my test failing, it should not check for user authentication.

Comment (leikkisa) [reply app/controllers/application_controller.rb]: Hey Sandra, yeah you should be able to take that out, it's already in verify_authentication. So you could remove the `if` altogether.  It feels like this shouldn't have caused your test to fail though, unless the user wasn't authenticated there, but still a good thing to catch!

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -4,9 +4,10 @@ class HomeController < ApplicationController
...
   def index
-    return redirect_to("/help") unless current_user&.authenticated?
-    return render("queue/index") if feature_enabled?(:case_search_home_page)
-
-    redirect_to("/help")
+    if current_user&.authenticated?
+      render("queue/index")
+    else
+      redirect_to("/help")
+    end
```
Comment (leikkisa) [inline app/controllers/home_controller.rb]: It'll look more like this:
```
return redirect_to("/help") unless current_user&.authenticated?
render("queue/index")
```

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (leikkisa) [reply app/controllers/application_controller.rb]:
```
return false if current_user.admin?
return false if current_user.organization_queue_user? || current_user.vso_employee?
return false if current_user.attorney_in_vacols? || current_user.judge_in_vacols?
return false if current_user.colocated_in_vacols?

true
```

File: app/controllers/home_controller.rb (hunk 2 above)
Comment (Sjones352) [reply app/controllers/home_controller.rb]: Fixed

Comment (Sjones352) [reply app/controllers/home_controller.rb]: Thanks for the suggestions Sally. Right now all tests are passing

**Pull request merged**